src/main/java/org/broadinstitute/hellbender/engine/VariantWalkerBase.java

```diff
@@ -129,6 +130,13 @@ protected VariantFilter makeVariantFilter() {
         return VariantFilterLibrary.ALLOW_ALL_VARIANTS;
     }
 
+    /**
+     * Returns the list of intervals to iterate, either limited to the user-supplied intervals or the entire reference genome if none were specified
+     */
+    public List<SimpleInterval> getIntervals() {
```

Comment:
sorry, somehow missed this.  I moved to GATKTool; however, I'm not sure if you're asking for more.  It already returns intervalsForTraversal directly if non-null; however, as the javadoc says the whole point of the method is to default back to the reference dictionary intervals if no user-supplied intervals are present.

because we're moving this to GATKTool and there is now the possibility of a walker where getReference() is false, I added a check for that situation.

Reply:
Also, I renamed the method to getEffectiveIntervals() to more clearly differentiate its purpose and differentiate from other basic getters.